source/code/plugin/health/health_container_cpu_memory_aggregator.rb

```diff
@@ -163,11 +175,24 @@ def get_records
             container_cpu_memory_records = []
 
             @cpu_records.each{|resource_key, record|
+
+                cpu_limit_mc = 1.0
+                if record["limit"].is_a?(Numeric)
```

Comment:
what happens if record["limit"] is nil?

Reply:
it will still use the value as 1.0

Reply:
wont this throw an exception in that case?